content/60.methods.md

```diff
@@ -57,40 +70,27 @@ By recording the evolution of information over time and assembling a resource th
+We sought to tag potential reviewers based on the introductions they had contributed in order to encourage participation.
+The emphasis on issues and pull requests was designed to encourage authors to discuss papers in the issues and to provide feedback (both formal and informal) on proposed text additions or changes.
+We also used gitter [@url:https://gitter.im] to promote informal questions and sharing of information among collaborators.
 
 #### Applying Manubot's Existing Capabilities to the Challenges of COVID-19 
 
 Because of the ever-evolving nature of the COVID-19 crisis, many of the figures and text proposed by subject matter contributors would have quickly become outdated.
-To address this concern, Manubot and GitHub's continuous integration features were used to create figures and text that could respond to changes in the COVID-19 pandemic over time.
-The combination of Manubot and GitHub Actions also made it possible to dynamically update information such as statistics and visualizations in the manuscript.
-When scientific writers added text that was current only as of a given date, publicly available data sources were identified whenever possible to allow the information to pulled directly into the manuscript in order to keep it up to date.<!--To Do: Add quoted examples-->
-Data was pulled from a number of sources.
-Data about worldwide cases and deaths from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series] were read using a Python script. <!-- To Do: replace figure reference with manuscript reference or reference figure conditionally based on template variable [to generate Figure @fig:csse-deaths.] -->
-Similarly, the clinical trials statistics and figure were generated based on data from the University of Oxford Evidence-Based Medicine Data Lab's COVID-19 TrialsTracker [@doi:10.5281/zenodo.3732709]. <!-- TODO: replace figure reference with manuscript reference or reference figure conditionally based on template variable [Figure @fig:ebm-trials] -->
-In both cases, frequency data were plotted using Matplotlib [@doi:10.1109/MCSE.2007.55] in Python.
-The figure showing the geographic distribution of COVID-19 clinical trials was generated using the countries associated with the trials listed in the COVID-19 TrialsTracker, converting the country names to 3-letter ISO codes using pycountry or manual adjustment when necessary, and visualizing the geographic distribution of trial recruitment using geopandas. <!-- TODO: replace figure reference with manuscript reference or reference figure conditionally based on template variable [Figure @fig:ebm-map] -->
-<!--To Do: Update with Vaccines-->
-
+To address this concern, Manubot and GitHub's CI features were used to create figures that integrated online data sources to respond to changes in the COVID-19 pandemic over time.
```

Comment:
I see this as a new feature.  Previously Manubot could only include static images, including versioned static images from an external GitHub repository.

Reply:
Oh wow! I didn't realize this was the first time this was done. I'll move it down in my next commit.